### PR TITLE
cut 2.12 for global hub

### PR DESCRIPTION
**Pull request by yanmxa**
Signed-off-by: myan <myan@redhat.com>

### CODE REVIEW

File: ci-operator/config/stolostron/multicluster-global-hub/stolostron-multicluster-global-hub-release-2.12.yaml [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+      REGISTRY_ORG: stolostron
+    workflow: ocm-ci-image-mirror
+- as: release-212-agent-image-mirror
+  postsubmit: true
+  steps:
+    dependencies:
+      SOURCE_IMAGE_REF: multicluster-global-hub-agent
+    env:
+      IMAGE_REPO: multicluster-global-hub-agent
+      IMAGE_TAG: v1.2.0
```
Comment (clyang82) [inline ci-operator/config/stolostron/multicluster-global-hub/stolostron-multicluster-global-hub-release-2.12.yaml:192]: should be `v1.3.0`

File: ci-operator/config/stolostron/multicluster-global-hub/stolostron-multicluster-global-hub-release-2.11.yaml [hunk 2]
```diff
@@ -128,7 +128,7 @@ tests:
     make setup_envtest
     export KUBEBUILDER_ASSETS="$(${TMP_BIN}/setup-envtest use --use-env -p path)"
     export SELF="make -f /opt/build-harness/Makefile.prow"
-    export SONAR_GO_TEST_ARGS="-coverpkg=./... ./pkg/... ./operator/... ./agent/... ./manager/..."
+    export SONAR_GO_TEST_ARGS="-coverpkg=./... ./pkg/... ./operator/... ./agent/... ./manager/... ./test/integration/..."
```
Comment (clyang82) [inline ci-operator/config/stolostron/multicluster-global-hub/stolostron-multicluster-global-hub-release-2.11.yaml:131]: we do not have `./test/integration/...` folder in release-2.11. do not need to modify here.

Comment (yanmxa) [reply ci-operator/config/stolostron/multicluster-global-hub/stolostron-multicluster-global-hub-release-2.11.yaml:131]: We have added some integration cases in the 2.11

File: ci-operator/config/stolostron/multicluster-global-hub/stolostron-multicluster-global-hub-release-2.12.yaml (hunk 1 above)
Comment (yanmxa) [reply ci-operator/config/stolostron/multicluster-global-hub/stolostron-multicluster-global-hub-release-2.12.yaml]: Done!

File: ci-operator/config/stolostron/multicluster-global-hub/stolostron-multicluster-global-hub-release-2.11.yaml (hunk 2 above)
Comment (yanmxa) [reply ci-operator/config/stolostron/multicluster-global-hub/stolostron-multicluster-global-hub-release-2.11.yaml]: oh, sorry, It's been corrected now.

**Review by clyang82 — APPROVED** [review]
LGTM

**Pull request merged**